packages/ui/src/i18n/dictionaries/authenticator/fr.ts

```diff
@@ -58,14 +58,30 @@ export const frDict: AuthenticatorDictionary = {
   'Your code is on the way. To log in, enter the code we sent you': `Votre code est en cours d'envoi. Pour vous connecter, saisissez le code que nous vous avons envoyé`,
 
   // Additional translations provided by customers
+  'Add your Profile': 'Ajoutez votre profil',
+  'Add your Website': 'Ajoutez votre site web',
   'An account with the given email already exists.':
     'Un utilisateur avec cette adresse email existe déjà.',
+  'Birthdate': 'Date de naissance',
   Change: 'Modifier',
   'Confirm a Code': 'Confirmer un code',
   'Confirm Sign In': 'Confirmer la connexion',
   'Create account': 'Créer un compte',
+  'Enter your Birthdate': 'Saisissez votre date de naissance',
+  'Enter your Confirmation Code': 'Saisissez votre code de confirmation',
+  'Enter your Email': 'Saisissez votre adresse email',
```

Comment:
This is no longer needed because we fixed a bug where the previous translation was lowercase `email`. Please pull this branch and remove this line.